conf/locale/fr/LC_MESSAGES/djangojs.po

```diff
@@ -8663,7 +8663,7 @@ msgstr "Réinitialiser mon mot de passe"
 
 #: lms/templates/student_account/register.underscore
 msgid "Already have an {platformName} account?"
-msgstr ""
+msgstr "Vous possédez déjà un compte {platformName} ?"
```

Comment:
This string is not being translated. Since this a javascript translation you need to run `python manage.py lms compilejsi18n` in order to generate static translation javascript files. Please run that and commit the resulting files for french.

Reply:
@xitij2000, sorry I wasn't aware of this and hence missed it. When I ran the command, it updated files for other languages as well and even in the file for French, a lot of additional changes are added. Do I have to commit only the changes corresponding to this line?

Reply:
I passed the `--locale fr` flag to restrict the updates to the French language only. But there are still additional changes that have been made. I guess since these are generated from the existing translation files already in Ironwood, it is okay to add them as-is.